applications/ecallmgr/src/ecallmgr_channel_move.erl

```diff
@@ -77,10 +77,9 @@ resume(UUID, NewNode, Evt) ->
 %% and its corresponding '>' to %3C and %3E as they should be
 -spec fix_metadata(ne_binary()) -> ne_binary().
 fix_metadata(Meta) ->
-    Replacements = [
-                    {<<"\<sip\:">>, <<"%3Csip:">>}
-                    ,{<<"\>\<sip">>, <<"%3E<sip">>}
-                    ,{<<"\>;">>, <<"%3E;">>} % this is especially nice :)
+    Replacements = [{<<"<sip\:">>, <<"%3Csip:">>}
```

Comment:
@fenollp How did you test that this works as expected? This code has been untouched since 2013 so I am hesitant to accept this commit without proof of testing. Show your work! :)